ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -190,6 +190,15 @@ open class DrawerController: UIViewController, TokenizedControlInternal {
             _contentView = newValue
             if let contentView = _contentView {
                 containerView.addArrangedSubview(contentView)
+                // Need to set constraints to prevent view getting cut off from UIPopoverArrowDirection in iPad.
+                if !showsResizingHandle {
+                    NSLayoutConstraint.activate([
+                        contentView.topAnchor.constraint(equalTo: containerView.safeAreaLayoutGuide.topAnchor),
+                        contentView.leadingAnchor.constraint(equalTo: containerView.safeAreaLayoutGuide.leadingAnchor),
+                        contentView.bottomAnchor.constraint(equalTo: containerView.safeAreaLayoutGuide.bottomAnchor),
+                        contentView.trailingAnchor.constraint(equalTo: containerView.safeAreaLayoutGuide.trailingAnchor)
```

Comment:
Seems odd to need all these constraints in a stack view. Does turning on layoutMarginsRelativeArrangement / insetsLayoutMarginsFromSafeArea help?

Reply:
your first suggestion worked, thanks!